src/libraries/Common/src/Interop/Windows/IpHlpApi/Interop.ConvertInterfaceNameToLuid.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class IpHlpApi
+    {
+        [LibraryImport(Libraries.IpHlpApi, SetLastError = true, StringMarshalling = StringMarshalling.Utf16, EntryPoint = "ConvertInterfaceNameToLuidW")]
+        internal static unsafe partial uint ConvertInterfaceNameToLuid(ReadOnlySpan<char> unicodeNullTerminatedName, ref ulong interfaceLuid);
```

Comment:
For all of these imports, we generally prefer to name the parameters exactly as they are in the native definition. If you want to add XML comments describing what's expected, that'd be fine.